internal/server/coap/coap.go

```diff
@@ -0,0 +1,66 @@
+package coap
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/mainflux/mainflux/internal/server"
+	"github.com/mainflux/mainflux/logger"
+	gocoap "github.com/plgd-dev/go-coap/v2"
+	"github.com/plgd-dev/go-coap/v2/mux"
+)
+
+const (
+	stopWaitTime = 5 * time.Second
```

Comment:
For instance in coap grpc and http sub packages all have 5 seconds stop time. why not have it in `internal/sever` package and all the sub packages to use it

Reply:
Does it make sense to extract it as an exported constant and use it in all the services (since there is no reason for using different values for graceful shutdown between services)?